apps/roboshield/src/components/RichText/styleConverter.tsx

```diff
@@ -0,0 +1,48 @@
+import { JSXConverters } from "@payloadcms/richtext-lexical/react";
+import { RichTypography, Link } from "@commons-ui/next";
+
+const DEFAULT_PROPS = {
+  html: false,
+};
+
+export const styleConverter = (converterProps: any): JSXConverters<any> => ({
+  heading: ({ node, nodesToJSX }) => {
+    const Tag = node.tag || "h1";
```

Comment:
Why are we defaulting to `h1`? Shouldn't we just leave it up to the default component in `RichTypography`?